### ISSUE TITLE
Problem with signed Message

### ISSUE THREAD

**Issue by ghsolutions** [PHPMailer/PHPMailer#821]
Hi, i send an email with phpmailer and signed it with a s/mime certificate. When i try to send it via Applemail all works fine, but when i send an email with phpmailer the receiver says the mail was signed but it was maybe changed after signing and it would not be save. 

I followed the Issue 376 and use the extra_cacertfile:

``` php
$mail = new PHPMailer();
$mail->isMail();
$mail->setFrom('root@test.de', 'root@test.de');
$mail->addReplyTo('root@test.de', 'root@test.de');
$mail->addAddress('info@test.de', 'GHD');
$mail->Subject = 'PHPMailer SMIME test';

$_msg = "Test";
$mail->msgHTML($_msg);
//Replace the plain text body with one created manually
$mail->AltBody = nl2br($_msg);

//Configure message signing (the actual signing does not occur until sending)
$mail->sign(
    'cert.pem', //The location of your certificate file
    'key.pem', //The location of your private key file
    'test',
    'certchain.pem' //The location of your chain file
);

$mail->send();
```

![voila_capture-2016-09-06_05-50-03_pm](https://cloud.githubusercontent.com/assets/81561/18760808/e0c397f8-8102-11e6-9e05-97c6530fa215.png)

I use PHP 5.6 on a Centos-Server. I used PHPMailer 5.2.14 and 5.2.16

**Comment by Synchro** [comment]
Does it work if you send via SMTP to localhost? Try this instead of `isMail()`:

```php
$mail->isSMTP();
$mail->Host = 'localhost';
```

**Comment by ghsolutions** [comment]
Hi, sorry i hadn't much time thw last weeks. Today i tried your tip and it works very well. thank you.

**Comment by ooxi** [comment]
Could you describe why this solves the signature failure @Synchro? I don't understand

**Comment by Synchro** [comment]
It appears that the `mail()` function reformats outbound messages unnecessarily which can break things, specifically header lines, see #818 for an example, and I suspect others may be symptoms of the same problem. There is a bug underlying this in #36, however, `mail()` should not be doing this because the messages are RFC compliant - it's being overly aggressive.

Sending via SMTP to localhost avoids the problem by not using mail(), so the reformatting does not happen.

Fixing #36 will help, but it's not fixable without a BC break (it needs the header label to be pased into `encodeHeader`, not just its value), so any fix would have to go into 6.0.
